### PR TITLE
Fix SpatialView latency.

### PR DESCRIPTION
**Pull request by samiwh**
#### Description
Added a missing line that was meant SpatialView had great bandwidth usage and terrible latency.
Set flushing to be default enabled.
Clang format changes to the settings header.

#### Release note
Flags involved are all experimental so none.

### CODE REVIEW

File: SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp [hunk 1]
```diff
@@ -101,7 +101,7 @@ USpatialGDKSettings::USpatialGDKSettings(const FObjectInitializer& ObjectInitial
 	, bTcpNoDelay(false)
 	, UdpServerDownstreamUpdateIntervalMS(1)
 	, UdpClientDownstreamUpdateIntervalMS(1)
-	, bWorkerFlushAfterOutgoingNetworkOp(false)
+	, bWorkerFlushAfterOutgoingNetworkOp(true)
```
Comment (m-samiec) [inline SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp:104]: Turning this on for the legacy connection changes the performance characteristics - can we just enable it when using spatial view?

Comment (samiwh) [reply SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp:104]: Sure

Comment (samiwh) [reply SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp:104]: Actually in that case it's probably better to just make MaybeFlush always flush in spatial view.
I'm happy to try and do something more elegant although the shim removal stuff changes all of this anyway.

Comment (m-samiec) [reply SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp]: Nice

**Pull request merged**